src/platform_impl/linux/x11/util/icon.rs

```diff
@@ -27,7 +27,7 @@ impl Icon {
         data.push(rgba_icon.height as Cardinal);
         let pixels = rgba_icon.rgba.as_ptr() as *const Pixel;
         for pixel_index in 0..pixel_count {
-            let pixel = unsafe { &*pixels.offset(pixel_index as isize) };
+            let pixel = unsafe { &*pixels.add(pixel_index) };
```

Comment:
This one scared me for a second (thought it was suddenly trying to use integer addition, which is strictly not the same as pointer offset), but `pointer::add` is just a convenience for this exact case